.rubocop_local.yml

```diff
@@ -0,0 +1,3 @@
+AllCops:
+  Exclude:
+  - db/schema.rb
```

Comment:
Since this is not a Rails app we don't have this directory.  Follow the pattern from the persister repo and just leave this as a blank file: https://github.com/RedHatInsights/topological_inventory-persister/blob/master/.rubocop_local.yml